liquid-rust-syntax/src/lexer.rs

```diff
@@ -11,6 +11,7 @@ use rustc_span::{symbol::kw, BytePos, Symbol};
 pub enum Token {
     Caret,
     EqEq,
+    Eq,
```

Comment:
I think the problem is that you have to add a case to `Cursor::map_token` to add new tokens.

Reply:
Ah yes, that fixes it!